resources/texts/_contentManager.html

```diff
@@ -359,13 +374,20 @@
             {{ book.tags }}
           </span>
           <span class="tablecell cell5">
-            <template v-if="book.downloadId">
-              <span v-if="downloads[book.id]">
+            <template v-if="downloads[book.id]">
+              <span>
                 {{ niceBytes(downloads[book.id].completedLength) }} / {{ niceBytes(downloads[book.id].totalLength) }}
               </span>
             </template>
-            <button v-else-if="book.path" v-on:click="openBook(book)">Open</button>
-            <button v-else v-on:click="downloadBook(book)">Download</button>
+          </span>
+          <span class="tablecell cell5">
+            <button v-if="book.path" v-on:click="openBook(book)">Open</button>
+            <button v-if="!book.path && !downloads[book.id]" v-on:click="downloadBook(book)">Download</button>
+            <button v-if="downloads[book.id] && downloads[book.id].status == 'active'" v-on:click="pauseBook(book)">Pause</button>
+            <button v-if="downloads[book.id] && downloads[book.id].status == 'paused'" v-on:click="resumeBook(book)">Resume</button>
+          </span>
+          <span v-if="downloads[book.id]" class="tablecell cell5">
+            <button v-on:click="cancelBook(book)">Cancel</button>
```

Comment:
The `div` "bookTable" is a table.It as 6 columns.
The header of the table is declared in the `header div`.
Each `span` represent one cell of the table. So it can't be two (or three) `span` with the class `cell5`.
Here, there are the span at line 376, the span at line 383 and (if a downoald is present) the span at line 389.
All the content of those three span should be merge in one.
Instead of having something like :
```html
<span class="tablecell cell5">
  <template v-if="downloads[book.id]" > ...</template>
</span>
<span class="tablecell cell5">
  <button v-if "book.path"> ...</button>
  <button v-if "!book.path"> ...</button>
  ...
</span>
<span v-if="downloads[book.id]" class="tablecell cell5">
  <button v-on:click="cancelBook(book)">Cancel</button>
</span>
```

We should have :
```html
<span class="tablecell cell5">
  <template v-if="downloads[book.id]" >
    ...
    <button v-on:click="cancelBook(book)">Cancel</button>
  </template>
  <button v-if "book.path"> ...</button>
  <button v-if "!book.path"> ...</button>
  ...
</span>
```